src/curve_client.cpp

```diff
@@ -240,6 +240,7 @@ int zmq::curve_client_t::process_ready (const uint8_t *msg_data_,
         session->get_socket ()->event_handshake_failed_protocol (
           session->get_endpoint (), ZMQ_PROTOCOL_ERROR_ZMTP_CRYPTOGRAPHIC);
         errno = EPROTO;
+        free(ready_plaintext);
```

Comment:
to fix the clang format error, add a space between the function name and the opening bracket, ditto below